### ISSUE TITLE
Cannot access AKS cluster with RBAC enabled

### ISSUE THREAD

**Issue by maximbaz** [databricks/click#81]
Hi there,

I wanted to try your project on my AKS cluster, but I cannot select anything except context due to missing access.

```
[none] [none] [none] > context dev
[dev] [none] [none] > pods
Server Error: pods is forbidden: User "clusterUser" cannot list pods at the cluster scope
```

Here's my `~/.kube/config` file:

```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: XXXX
    server: https://XXX:443
  name: dev
contexts:
- context:
    cluster: dev
    user: clusterUser_dev
  name: dev
current-context: dev
kind: Config
preferences: {}
users:
- name: clusterUser_dev
  user:
    client-certificate-data: XXX
    client-key-data: XXX
    token: XXX
```

**Comment by evenh** [comment]
I observe the same issues as well when trying to use click with a RBAC-enabled cluster

**Comment by nicklan** [comment]
Ohh hah, so AKS wants *both* a `Bearer` token, and the client cert/key.  I'll need to refactor the auth stuff a bit to make this work, but should have a fix in the near future.

**Comment by nicklan** [comment]
This is fixed by #96 in my testing.  Can any of you pull master, build and test for me that it works for you too?  Thanks!

**Comment by maximbaz** [comment]
Everything works perfectly, thanks @nicklan!
